color-eyre/src/config.rs

```diff
@@ -493,7 +492,7 @@ impl HookBuilder {
     ///
     /// ```rust
     /// color_eyre::config::HookBuilder::default()
-    ///     .panic_section("consider reporting the bug at https://github.com/yaahc/color-eyre")
+    ///     .panic_section("consider reporting the bug at https://github.com/eyre-rs/eyre")
```

Comment:
tiny ergo suggestion:
What do you think about giving folks a direct link to the issues page?
https://github.com/eyre-rs/eyre/issues/
We don't want issues/new because we want folks to see if it's been reported lately ofc. Might even help to mention, "include color-eyre in the title" -- maybe that's something that is better to go on the instructions of `issues/new` though.